backend_new/prisma/schema.prisma

```diff
@@ -0,0 +1,1025 @@
+  customMapPin                          Boolean?                      @map("custom_map_pin")
+  publishedAt                           DateTime?                     @map("published_at") @db.Timestamptz(6)
+  closedAt                              DateTime?                     @map("closed_at") @db.Timestamptz(6)
+  afsLastRunAt                          DateTime?                     @default(dbgenerated("'1970-01-01 00:00:00-07'::timestamp with time zone")) @map("afs_last_run_at") @db.Timestamptz(6)
+  lastApplicationUpdateAt               DateTime?                     @default(dbgenerated("'1970-01-01 00:00:00-07'::timestamp with time zone")) @map("last_application_update_at") @db.Timestamptz(6)
+  buildingAddressId                     String?                       @map("building_address_id") @db.Uuid
+  applicationPickUpAddressId            String?                       @map("application_pick_up_address_id") @db.Uuid
+  applicationDropOffAddressId           String?                       @map("application_drop_off_address_id") @db.Uuid
+  applicationMailingAddressId           String?                       @map("application_mailing_address_id") @db.Uuid
+  buildingSelectionCriteriaFileId       String?                       @map("building_selection_criteria_file_id") @db.Uuid
+  jurisdictionId                        String?                       @map("jurisdiction_id") @db.Uuid
+  leasingAgentAddressId                 String?                       @map("leasing_agent_address_id") @db.Uuid
+  reservedCommunityTypeId               String?                       @map("reserved_community_type_id") @db.Uuid
+  resultId                              String?                       @map("result_id") @db.Uuid
+  featuresId                            String?                       @unique() @map("features_id") @db.Uuid
+  utilitiesId                           String?                       @unique() @map("utilities_id") @db.Uuid
+  // START DETROIT SPECIFIC
+  hrdId                                 String?                       @map("hrd_id")
+  ownerCompany                          String?                       @map("owner_company")
+  managementCompany                     String?                       @map("management_company")
+  managementWebsite                     String?                       @map("management_website")
+  amiPercentageMin                      Int?                          @map("ami_percentage_min")
+  amiPercentageMax                      Int?                          @map("ami_percentage_max")
+  phoneNumber                           String?                       @map("phone_number")
+  temporaryListingId                    Int?                          @map("temporary_listing_id")
+  isVerified                            Boolean?                      @default(false) @map("is_verified")
+  marketingType                         MarketingTypeEnum             @default(marketing) @map("marketing_type")
```

Comment:
Should be optional considering core doesn't use this

Reply:
I think the default should be sufficient (if core doesn't use it whether its null or a defaulted value won't really make a difference)